scripts/linters/pylint_extensions_test.py

```diff
@@ -876,8 +879,79 @@ def test_no_newline_above_return(self):
             self.checker_test_object.checker.visit_functiondef(
                 node_with_no_space_above_return)
 
-    def test_varying_combination_of_newline_above_args(self):
-        node_newline_above_args_raises = astroid.extract_node(
+    def test_newline_below_multipart_docstring_section(self):
+        node_multipart_docstring_args = astroid.extract_node(
+            u"""def func(arg): #@
+                \"\"\"Return something in dict.
+
+                Args:
+                    arg: dict. Some sample
+                        dictionary.
+
+                Returns:
+                    dict. Sample dict.
+                    Example:
+
+                    {
+                        'entry1': {
+                            'number': 10,
+                        },
+                    }
+                \"\"\"
+                return something
+        """)
+
+        with self.checker_test_object.assertNoMessages():
+            self.checker_test_object.checker.visit_functiondef(
+                node_multipart_docstring_args)
+
+        node_multipart_varying_incorrect_spacing = astroid.extract_node(
+            u"""def func(arg): #@
+                \"\"\"Return something in dict.
+
+                Args:
+                    arg: dict. Some sample
+                        dictionary.
+
+
+                Returns:
+                    dict. Sample dict.
+                    Example:
+
+                    {
+                        'entry1': {
+                            'number': 10,
+                        },
+                    }
+
+                \"\"\"
+                return something
+        """)
+
+        empty_line_end_message = testutils.Message(
+            msg_id='empty-line-before-end',
+            node=node_multipart_varying_incorrect_spacing
+        )
+        message_above_returns = testutils.Message(
+            msg_id='single-space-above-returns',
```

Comment:
Do we already have these checks? IF yes, can we structure it better so that one one function cehks above and below newlines?

Reply:
@DubeySandeep Thanks for the review. Wanted to bring up an important point: currently, there are checks for empty spaces above each docstring section, and there is a separate check to ensure there is only a single newline at the end of each docstring. Thus, this new single space below check already seems a bit redundant (Single space below section #1 is equivalent to single space above section #2. Single space below last section is covered by single newline at end of docstring). I don't think it would make sense to merge these two. Please correct me if I'm wrong but perhaps this new lint check might not even be needed. What are your thoughts?

Reply:
Sorry for the misunderstanding, by "merge" I mean can we have one check which will ensure the newlines expectations.

Thanks for raising this point, can you confirm that the lint test will fail if we won't add a newline between any docstring section? If yes, then I agree that this new check would be redundant.

Reply:
Hi @DubeySandeep. Here is an example test showing the redundancy in question: 
![image](https://user-images.githubusercontent.com/15708812/100066013-5dd80480-2de9-11eb-8546-e040f23e57d2.png)

If you look at the messages, both `single-space-above-returns` and `single-space-below-args` refer to the same issue. Since this check is redundant, I can close this PR, and it would be a good idea to remove it from #8423 .